src/Model/QuestionManager.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace App\Model;
+
+class QuestionManager
+{
+    public const TABLE = '';
```

Comment:
No you can't have a Manager without a TABLE : it won't work. And we don't want to have not working code on dev